anta/device.py

```diff
@@ -171,12 +171,13 @@ async def collect_commands(self, commands: list[AntaCommand]) -> None:
         """
         await asyncio.gather(*(self.collect(command=command) for command in commands))
 
-    def supports(self, command: AntaCommand) -> bool:
+    def supports(self, command: AntaCommand, *, log: bool = True) -> bool:
```

Comment:
Any reason why using a boolean for logging and set it to True only once ?
I would suggest to remove this boolean and put the warning where it was :)